src/pages/buy/buy.layout.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { PopupContainer } from '@components/popup/container';
+import { Header } from '@components/header';
+import { Caption, Text, Title } from '@components/typography';
+import { Button, Flex, Stack } from '@stacks/ui';
+import { Link } from '@components/link';
+import AddFunds from '@assets/images/add-funds.svg';
+import { SpaceBetween } from '@components/space-between';
+import { openInNewTab } from '@common/utils/open-in-new-tab';
+import { ActiveFiatProviderType } from '@query/hiro-config/hiro-config.query';
+
+const PROVIDERS_INFO = {
+  transak: {
+    title: 'Transak',
+    body: 'International users can purchase STX quickly with EUR, KRW and many more.',
+    cta: 'Buy on Transak',
+  },
+  okcoin: {
+    title: 'Okcoin',
+    body: 'US users can purchase STX quickly with USD',
+    cta: 'Buy on Okcoin',
+  },
+};
+
+interface ProvidersUrl {
+  transak: string;
+  okcoin: string;
+}
+
+interface ProviderLayoutProps {
+  provider: string;
+  providerUrl: string;
+}
+
+const ProviderLayout = ({ provider, providerUrl }: ProviderLayoutProps) => {
```

Comment:
Our best practices asks for function declarations over expressions.

Reply:
Ok, yep, I did have a hard time converting a few of them ...but we could keep the standard for all that are not memoized, etc. I don't have a strong feeling about it on my end. I would just like to know if I am expected to refactor them and/or point it out in code review.